Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+ADD . /project
+WORKDIR /project
+
+RUN apt-get -y update
+RUN apt-get -y install postgresql-client
+RUN pip install -r requirements.txt
+
+CMD python src/manage.py runserver 0.0.0.0:8000
```

Comment:
@vxsx docker will run on it's own IP and will have no conflict with localhost. Yet it's still a todo for the future.